content/en/dashboards/functions/rollup.md

```diff
@@ -10,6 +10,10 @@ Every metric query is inherently aggregated. However, appending the `.rollup()`
 * The rollup `<interval>`: the interval of time your data is aggregated over ([if larger than the query-enforced rollup interval](#rollup-interval-enforced-vs-custom)).
 * The rollup `<aggregator>`: How your data points are aggregated within a given rollup time interval.
 
+To apply a rollup, you can navigate to the "Add function" (Σ) button while editing your query: 
+
+![rollup_nav](https://github.com/DataDog/documentation/assets/5199201/b3d04f2e-07c4-42d7-ac19-db6744a46f3d) 
```

Comment:
Are customers having trouble finding the Add function button or are they also having trouble finding the rollup option? Would the static image of the function button work?
```suggestion 
```

Reply:
Basically the motivation for this content is that the entire rollup section doesn't clarify how or where a user can add this function to their query, Kathy thought it would be prudent to clearly outline this UI flow for customers

Reply:
Yup, that makes sense, we could add more descriptive instructions. If you thinks a gif is necessary in addition to that, can you send me the file and I'll add it in a separate PR.